Automattic/jetpack
# tools: Update dependency update tooling

#### Changes proposed in this Pull Request:

* Add a script, tools/fixup-project-versions.sh, to fix "everything".
* Add a "-f" option to tools/changelogger-validate-all.sh to have it fix
  problems. Also reduce its default verbosity.
* Have tools/project-version.sh recommend the new script too when fixes
  are needed.
* When in non-verbose mode, have these tools display a spinner.

#### Jetpack product discussion


p1634646313162100-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


no

#### Testing instructions:






* Check out a commit that needs a bunch of package version and dependency updates, e.g. 3b406f19b5.
* Cherry pick this PR.
* Run `tools/fixup-project-versions.sh`. Make sure it updated everything needing updating.
